ansible/roles/postgresql/defaults/main.yml

```diff
@@ -162,7 +162,7 @@ postgresql__user_clusters: []
 
 # .. envvar:: postgresql__roles [[[
 #
-# List of PostgresSQL roles, specified as YAML dicts. See
+# List of postgresql roles, specified as YAML dicts. See
```

Comment:
The `PostgreSQL` spelling should be added to the dictionary. :-) And yeah, I see the additional `s` there now, I wonder if `codespell` will trigger when that's fixed. Just adding an earmark to check this later.